doc/tools.yml

```diff
@@ -187,3 +187,16 @@ z3:
     - 'formal:min'
     - 'formal'
     - 'formal:all'
+
+#---
+
+apicula:
+  url: 'https://hdl.github.io/awesome/items/apicula'
+  pkg:
+    - 'apicula'
+  use:
+    - 'apicula'
+  in:
+    - 'impl'
+  otherin:
+    - 'nextpnr:apicula'
```

Comment:
This seems not to be implemented. That is, `nextpnr.dockerfile` and `nextpnr.yml` are not updated in this PR.